backend/Makefile

```diff
@@ -7,31 +7,43 @@ build:
 
 run:
 	-pkill docker-compose
-	DATABASE=postgres docker-compose up
+	docker-compose up postgres backend
+	docker-compose up
 
 run_background:
 	-pkill docker-compose
-	DATABASE=postgres docker-compose up -d
+	docker-compose up -d postgres backend
 
 stop:
 	docker-compose down
 
+run_test:
+	docker build -t rafflebay/test .
+	docker-compose up -d postgres_test test 
+	docker-compose exec postgres_test psql -p 5433 -U postgres rafflebay_test -c "DROP TABLE bids, items, users CASCADE;"
+	docker-compose down
+
+
 test:
 	docker build -t rafflebay/test .
 	-pkill docker-compose
-	DATABASE=postgres_test docker-compose up
+	docker-compose up --abort-on-container-exit postgres_test test
+	docker-compose down	
 
 format:
 	npx prettier --write --arrow-parens always --single-quote --trailing-comma all --no-bracket-spacing "src/**/*.js"
 	npx prettier --write --arrow-parens always --single-quote --trailing-comma all --no-bracket-spacing "test/**/*.js"
 
 # Open shell to the postgres instance
 p_shell:
-	DATABASE=postgres docker-compose exec postgres psql -U postgres rafflebay
+	docker-compose exec postgres psql -U postgres rafflebay
+
+test_p_shell:
+	docker-compose exec postgres_test psql -U postgres -p 5433 rafflebay_test
 
 #CAUTION: WILL DELETE ALL DATA
 reset_db: 
-	DATABASE=postgres docker-compose exec postgres psql -U postgres rafflebay -c "DROP TABLE payments, shipments, bids, items, users;"
+	docker-compose exec postgres psql -U postgres rafflebay -c "DROP TABLE bids, items, users;"
```

Comment:
We need to also drop payments and shipments tables

Reply:
Fixed this